packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -3797,10 +3778,8 @@ function attemptEarlyBailoutIfNoScheduledUpdate(
       return updateOffscreenComponent(current, workInProgress, renderLanes);
     }
     case CacheComponent: {
-      if (enableCache) {
-        const cache: Cache = current.memoizedState.cache;
-        pushCacheProvider(workInProgress, cache);
-      }
+      const cache: Cache = current.memoizedState.cache;
```

Comment:
Need to fix these lints. Interesting that it wasn't failing for shadowing.

Reply:
TIL case statements without braces do not scope let/const to that case, fixed by adding braces to the previous case.